src/main/java/org/folio/rest/impl/base/AbstractEmail.java

```diff
@@ -196,12 +206,12 @@ private Future<Configurations> fetchSmtpConfigurationFromModConfig(OkapiClient o
       .compose(response -> {
         if (response.statusCode() == HTTP_OK.toInt()) {
           Configurations config = response.bodyAsJsonObject().mapTo(Configurations.class);
-          logger.info("Successfully fetched {} configuration entries", config.getConfigs().size());
+          logger.info("fetchSmtpConfigurationFromModConfig:: Successfully fetched {} configuration entries", config.getConfigs().size());
```

Comment:
We can log configs as a list, it's not going to be huge. Also, logList() is going to only log a limited amount of elements